src/components/geometry.js

```diff
@@ -143,6 +144,33 @@ function getGeometry (data, schema) {
     case 'plane': {
       return new THREE.PlaneBufferGeometry(data.width, data.height);
     }
+    case 'polygon': {
+      var verts = data.vertices.split(' ');
```

Comment:
similar to https://github.com/aframevr/aframe/blob/dev/src/core/propertyTypes.js#L42